app/models/like.rb

```diff
@@ -1,2 +1,10 @@
 class Like < ActiveRecord::Base
+  belongs_to :author, class_name: 'User'
+  belongs_to :post, class_name: 'Post'
```

Comment:
- [ ] OPTIONAL
It's redundant to include `class_name: 'Post'`  to _like_ association with _post_. Simply do something like:
```ruby
# app/model/like.rb
belongs_to :post